baseband/gsb/tests/test_gsb.py

```diff
@@ -436,6 +441,12 @@ def test_raw_stream(self, tmpdir):
             assert fh_r.header0.time == fh_r.start_time
             assert fh_r.header0 == frame1.header
             assert fh_r.size == 10 * fh_r.samples_per_frame
+            # Sample rates aren't exactly equal because GSB timestamps only
+            # store 9 digits of the fractional second.
+            assert np.all(np.isclose(fh_r.sample_rate, sample_rate,
+                                     rtol=1e-9, atol=0.*u.Hz))
+            assert np.isclose(fh_r._sample_rate, sample_rate.value,
```

Comment:
No testing of private properties unless they are really different from the public ones.